src/SonarScanner.Shim/PropertiesWriter.cs

```diff
@@ -284,6 +284,27 @@ private static bool IsAscii(char c)
             return c <= sbyte.MaxValue;
         }
 
+        internal /* for testing purposes */ string JoinAsSonarQubeMultiValueProperty(IEnumerable<string> paths)
+        {
+            if (Version.TryParse(this.config.SonarQubeVersion, out var sonarqubeVersion) &&
+                sonarqubeVersion.CompareTo(new Version(6, 5)) >= 0)
+            {
+                return string.Join(MultiValuesPropertySeparator, paths.Select(path => $"\"{path.Replace("\"", "\"\"")}\""));
```

Comment:
CSV like pattern.

Reply:
Can the SQ version ever include a non-version friendly suffix e.g. "-snapshot"? I seem to remember the plugins could, but we probably don't need to worry about it here.

Reply:
I wasn't sure hence this test.